### ISSUE TITLE
(Investigate) Apparent 404 received by user

### ISSUE THREAD

**Issue by alexellis** [openfaas/faas-netes#293]
## Expected Behaviour

User claims to receive 404 when invoking a function using OpenFaaS and gateway 0.9.2

I am unable to reproduce this in any environment including: GKE and kubeadm on Linux. My team cannot reproduce the issue on minikube / Docker for Mac @ivanayov 

## Steps to reproduce

Can't reproduce, but these are the steps:

* Install OpenFaaS using helm
* `faas-cli store deploy figlet`
* Invoke the figlet function

Reported by @LucasRoesler @dkozlov

**Comment by alexellis** [comment]
I just set up a brand new GKE cluster with Kubernetes 1.10.7-gke.1 and went through all the steps in HELM.md and chart/openfaas/README.md with a LB as my way of exposing the gw.

Everything worked perfectly including deploying figlet and invoking it.

I then made a Python3 / flask app using [this template](https://github.com/openfaas-incubator/python-flask-template): 

```
faas-cli deploy --image alexellis2/fl1:0.1 --name fl1 -g http://$OPENFAAS_URL
echo | faas-cli invoke  fl1 -g http://$OPENFAAS_URL
Test.. OK
```

**Comment by dkozlov** [comment]
Additional information: I am using NodePort and deploying function with --gateway http://127.0.0.1:31112 from kubernetes master

**Comment by stefanprodan** [comment]
Derek close

**Comment by stefanprodan** [comment]
This has been resolved in #299
